lib/icingadb/icingadb.hpp

```diff
@@ -81,6 +82,7 @@ class IcingaDB : public ObjectImpl<IcingaDB>
 	static String FormatCommandLine(const Value& commandLine);
 	static long long TimestampToMilliseconds(double timestamp);
 
+	static ArrayData GetObjectIdentifiersPerEnv(const ConfigObject::Ptr& object);
```

Comment:
I don't understand the `...PerEnv` suffix. The env doesn't have anything to do with this function? Maybe `...WithoutEnv` would be more fitting? 😄